README.md

```diff
@@ -1 +1,63 @@
-# oop-review
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# curriculum-cr-training-oop
+This Application is based on the Ruby OOP: school library projects 1 and 2.
+
+## Built With
+
+- Ruby
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
- [ ] It would be better to either add contents to these points or remove them when not needed for the project. For example; this project does not require us to have a test suite or deployment, so it can be removed. This will help us maintain a professional repository